src/runtime/pkg/device/manager/manager.go

```diff
@@ -195,7 +195,6 @@ func (dm *deviceManager) AttachDevice(ctx context.Context, id string, dr api.Dev
 	if !ok {
 		return ErrDeviceNotExist
 	}
-
```

Comment:
Do we need to add this change to the commit? Thanks.

Reply:
Of course not. Committed but not pushed.

Reply:
Waiting for the static-checks to complete.